noxfile.py

```diff
@@ -114,4 +114,12 @@ def copy_docker_db_config_templates(session: nox.Session):
         shutil.rmtree(target_path)
     with session.chdir(ROOT):
         session.run("cp", "-rL", "docker_db_config_template", str(target_path))
-    session.run("git", "add", str(target_path))
\ No newline at end of file
+    session.run("git", "add", str(target_path))
+
+@nox.session(name="lint:security", python=False)
```

Comment:
I guess this is a hotpatch, please create a ticket that it gets replaced

Reply:
Actually, Nico already fixed it in version 0.18.0. Will update dependencies and try again without it